TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/Status.cshtml.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
+
+[Journey(JourneyNames.EditInduction), ActivatesJourney, RequireJourneyInstance]
+public class StatusModel : CommonJourneyPage
+{
+    protected TrsDbContext _dbContext;
+
+    [BindProperty]
+    public InductionStatus InductionStatus { get; set; }
+
+    public InductionJourneyPage NextPageInJourney
+    {
+        get
+        {
+            return InductionStatus switch
+            {
+                InductionStatus.Exempt => InductionJourneyPage.ExemptionReason,
+                _ when InductionStatus.RequiresStartDate() => InductionJourneyPage.StartDate,
+                _ => InductionJourneyPage.ChangeReason
+            };
```

Comment:
Is it the intention that InductionStatus contain the business logic in terms of what data is needed and the views just route accordingly? If InductionStatus had a RequiresExemptionReason() then I could do that.